src/wallet/wallet2.cpp

```diff
@@ -4398,6 +4398,8 @@ void wallet2::clear_user_data()
     i->second.m_dests.clear();
   for (auto i = m_unconfirmed_txs.begin(); i != m_unconfirmed_txs.end(); ++i)
     i->second.m_dests.clear();
+  for (auto i = m_transfers.begin(); i != m_transfers.end(); ++i)
+    i->m_frozen = false;
```

Comment:
What is this good for, why would it be important to unfreeze?

Reply:
Freezing outputs is a feature that lets users manually prevent the wallet from spending specific outputs. Users can't spend from background [syncing] wallets, so an output's frozen status is extra user-provided data that doesn't need to be saved to the background wallet.

Reply:
Ah, I see, there is no technical problem, just the issue of completely removing *any* user-provided data from a background wallet. Neat.